docs/conf/scalar-prometheus-custom-values.yaml

```diff
@@ -34,6 +39,25 @@ grafana:
       # -- https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding
       allow_embedding: false
 
+  # -- Additional data source configurations
+  additionalDataSources:
+  - name: Prometheus
+    type: prometheus
+    uid: prometheus
+    url: http://scalar-monitoring-kube-pro-prometheus.monitoring.svc.cluster.local:9090/
+    access: proxy
+    editable: false
+    isDefault: false
+    jsonData:
+      timeInterval: 30s
```

Comment:
This is a configuration for the Prometheus data source. It's based on the default configuration of the kube-prometheus-stack. However, I set `isDefault = false` to avoid some errors.